### ISSUE TITLE
Revisit roles and ARIA attributes used in accordions

### ISSUE THREAD

**Issue by cjg89** [UCF/Athena-Framework#235]
**Describe the bug**

Our current usage and examples for accordions utilize a tab design pattern (role = tablist, tab, tabpanel).  Ideally, we should be following the accordion design pattern:
- https://www.w3.org/TR/wai-aria-practices-1.1/#accordion
- https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html

Particularly, the tab pattern for accordions that we use now is problematic because we nest interactive elements (the toggle button/link for each collapsible panel within the `.card-header`s with `role="tab"`.)

**Steps to Reproduce**

See Athena documentation for accordions.

**Expected Behavior**

Our accordions should follow the recommended design pattern for accordions, and should not utilize nested interactive elements.

**Possible Fix**

Update examples in documentation, and potentially modify collapse javascript to accommodate needed changes.

**Your environment:**

n/a

**Additional context**

n/a
